project/src/js/ui/tournament/detail/tournamentDetailView.js

```diff
@@ -182,23 +182,48 @@ function timeInfo(time, preceedingText) {
 
 function tournamentLeaderboard(ctrl) {
   const data = ctrl.tournament();
+  const players = data.standing.players;
+  const page = data.standing.page;
+  const firstPlayer = (players.length > 0) ? players[0].rank : 0;
+  const lastPlayer = (players.length > 0) ? players[players.length-1].rank : 0;
+  const backEnabled = page > 1;
+  const forwardEnabled = page < data.nbPlayers/10;
+  const isUserPage = data.me && (page === Math.ceil(data.me.rank/10));
+  const user = session.get();
+  const userName = user ? user.username : '';
+
   return (
     <div key="leaderboard" className='tournamentLeaderboard'>
       <p className='tournamentTitle'> {i18n('leaderboard')} ({data.nbPlayers} Players)</p>
+
       <table className='tournamentStandings'>
-        {data.standing.players.map(renderLeaderboardItem)}
+        {data.standing.players.map(createLeaderboardItemRenderer(userName))}
```

Comment:
Thanks for the mini lesson on bind. Will use in the future where it makes sense. Committed this change and pushed it.